CMakeLists.txt

```diff
@@ -150,12 +150,12 @@ else()
 endif()
 
 # Can't use the `REUSE_FROM` option bc tt_lib and ttnn have different build flags :(
-add_library(pch_pybinds INTERFACE)
-target_precompile_headers(pch_pybinds INTERFACE
-    ${PROJECT_SOURCE_DIR}/tt_metal/third_party/pybind11/include/pybind11/operators.h
-    ${PROJECT_SOURCE_DIR}/tt_metal/third_party/pybind11/include/pybind11/pybind11.h
-    ${PROJECT_SOURCE_DIR}/tt_metal/third_party/pybind11/include/pybind11/stl.h
-)
+# add_library(pch_pybinds INTERFACE)
```

Comment:
PCH to be disabled for now. Due to addition of a link between tt_eager and ttnn_lib something causes pch to get updated in the middle of the build. Will turn PCH back on when tt_eager and ttnn are merged.

Reply:
Linking issue here
https://github.com/tenstorrent/tt-metal/issues/9457

Reply:
Might wanna leave a comment within the `CMakeLists.txt` with a link to that issue so it doesn't get lost

Reply:
Already opened a PR fixing this